DOC-853: Credential node overhaul E-G (except Google) (#2090)
n8n-io/n8n-docs: docs/integrations/builtin/credentials/erpnext.md

Refer to [ERPNext's developer documentation](https://frappeframework.com/docs/user/en/introduction){:target=_blank .external-link} for more information about working with the framework.

## Using API key

To configure this credential, you'll need:

- An **API Key**: Generate this from your own ERPNext user account in **Settings > My Settings > API Access**.
- An **API Secret**: Generated with the API key.
- Your ERPNext **Environment**:
    - For **Cloud-hosted**:
        - Your ERPNext **Subdomain**: Refer to the [FAQs](#how-to-find-the-subdomain-of-an-erpnext-cloud-hosted-account)
        - Your **Domain**: Choose between `erpnext.com` and `frappe.cloud`.
    - For **Self-hosted**:
        - The fully qualified **Domain** where you host ERPNext
- Choose whether to **Ignore SSL Issues**: When selected, n8n will connect even if SSL certificate validation is unavailable.

If you are an ERPNext System Manager, you can also generate API keys and secrets for other users. Refer to the [ERPNext Adding Users documentation](https://docs.erpnext.com/docs/user/manual/en/adding-users){:target=_blank .external-link} for more information.

> ## FAQs

Reviewer: ```suggestion

```

> ### How do I find the subdomain of an ERPNext cloud-hosted account?

Reviewer: ```suggestion
## How to find the subdomain of an ERPNext cloud-hosted account
```
